modules/040-node-manager/templates/capi-controller-manager/deployment.yaml

```diff
@@ -0,0 +1,183 @@
+{{- define "capi_controller_resources" }}
+cpu: 10m
+memory: 50Mi
+{{- end }}
+
+{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+---
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: capi-controller-manager
+  namespace: d8-cloud-instance-manager
+  {{- include "helm_lib_module_labels" (list . (dict "app" "capi-controller-manager" "workload-resource-policy.deckhouse.io" "master")) | nindent 2 }}
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind: Deployment
+    name: capi-controller-manager
+  updatePolicy:
+    updateMode: "Auto"
+  resourcePolicy:
+    containerPolicies:
+    - containerName: "capi-controller-manager"
+      minAllowed:
+        {{- include "capi_controller_resources" . | nindent 8 }}
+      maxAllowed:
+        cpu: 20m
+        memory: 70Mi
+    {{- include "helm_lib_vpa_kube_rbac_proxy_resources" . | nindent 4 }}
```

Comment:
PDB and CAPI Controller Manager will deploy only when vertical-pod-autoscaler-crd is enabled.
I think, we need move `{end}` block from [end of file](https://github.com/deckhouse/deckhouse/pull/5129/files#diff-2b13e48b845159546e35cf8a4e44fe752a20097c0254f108ee331083fc008cd1R183) to this location, for example like [here](https://github.com/deckhouse/deckhouse/blob/e834bcd452f75193f0228a97b11b6c398e745398/ee/modules/030-cloud-provider-openstack/templates/cloud-data-discoverer/deployment.yaml#L5C1-L31C4)

Reply:
yep, it is mistake, thanks !